Option toc_hover=False raises Theme error

When I use this theme with `toc_hover=False`, I get the following error
```
Theme error:
An error happened in rendering the page _autosummary/[filename]
Reason: UndefinedError("'dropdown_link' is undefined")
make: *** [Makefile:60: html] Error 2
```
Searching for `dropdown_link` in this repo shows that this variable is only conditionally defined in the template.

--- a/.github/CHANGELOG.md
+++ b/.github/CHANGELOG.md
@@ -4,6 +4,11 @@
 
 * Bumped `wheel` to v0.38.1. [(#33](https://github.com/XanaduAI/xanadu-sphinx-theme/pull/33)
 
+### Bug fixes
+
+* Fixed an issue where the `dropdown_link` Jinja variable is accessed but undefined.
+  [#36](https://github.com/XanaduAI/xanadu-sphinx-theme/pull/36)
+
 ### Contributors
 
 This release contains contributions from (in alphabetical order):

--- a/xanadu_sphinx_theme/_version.py
+++ b/xanadu_sphinx_theme/_version.py
@@ -3,4 +3,4 @@ This module specifies the Xanadu Sphinx Theme version with https://semver.org/
 using the following format: <major>.<minor>.<patch>[-<pre-release>].
 """
 
-__version__ = "0.3.6"
+__version__ = "0.4.0-dev"

--- a/xanadu_sphinx_theme/header.html
+++ b/xanadu_sphinx_theme/header.html
@@ -43,6 +43,8 @@
 {% if use_hover_toc and theme_navbar_left_links|length != 0 %}
   {# Extract active navbar entries #}
   {% set dropdown_link = theme_navbar_left_links|selectattr("active") | map(attribute="name") | list %}
+{% else %}
+  {% set dropdown_link = [] %}
 {% endif %}
 
 {% if use_hover_toc and dropdown_link|length == 0 %}

